.github/workflows/linux.yml

```
@@ -67,7 +67,10 @@ jobs:
     runs-on: [ubuntu-22.04]
     container:
       image: dealii/dependencies:jammy
+      options: --user root
```

Another issue as reported [here](https://github.com/dealii/dealii/actions/runs/5126288407/jobs/9220596917).
```
mpiexec has detected an attempt to run as root.
```
In the current state of #15266, they overwrote the behavior with the following lines:
```suggestion
      options: --user root --env OMPI_ALLOW_RUN_AS_ROOT=1 --env OMPI_ALLOW_RUN_AS_ROOT_CONFIRM=1
```
Let's give this a try...